ahnlich/Makefile

```diff
@@ -21,7 +33,23 @@ check: ## cargo check
 test: ## cargo test
 	cargo nextest run --no-capture
 
-all: format check test clippy ## format, clip and test code
+
+bumpversion: ## Bump project versions. Rules for bumpversion: patch, minor, major.
```

Comment:
Let us rename this to make it more obvious that this is bumping the network protocol version. Or atleast that's what I think it does